site.region

```diff
@@ -38,10 +38,10 @@
       </a>
       <nav class="site-header__mediumScreenNavContainer" role="navigation">
         <ul role="listbox" class="site-header__mediumScreenNav boldSansSerif">
-          <li><a href="https://red-badger.com/what-we-do/">What we do</a></li>
-          <li><a href="https://red-badger.com/about-us/">About us</a></li>
-          <li><a class="activeNavLink" href="http://red-badger.com/blog/">Blog</a></li>
-          <li><a href="https://red-badger.com/about-us/events/">Events</a></li>
+          <li><a href="https://red-badger.com/what-we-do/" onclick="ga('send', 'event', 'DesktopHeaderNavigation', 'What we do', 'From: /blog')">What we do</a></li>
```

Comment:
As mentioned in person, it would be nice to have this on a new line so the maximum line length is 80